src/main/java/com/ebay/myriad/scheduler/TaskFactory.java

```diff
@@ -8,16 +9,10 @@
 import com.google.common.base.Strings;
 import com.google.gson.Gson;
 import com.google.protobuf.ByteString;
+import org.apache.hadoop.yarn.conf.YarnConfiguration;
 import org.apache.mesos.Protos;
-import org.apache.mesos.Protos.CommandInfo;
+import org.apache.mesos.Protos.*;
```

Comment:
Nit: Again, I really hate seeing * imports. Can we please be explicit about our imports?